docs/pages/admin-guides/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -329,6 +329,15 @@ resource allowed by "root-node-access" they will be required to provide a reason
 role set includes multiple roles governing Access Requests to the same roles and resources,
 "require" mode takes precedence.
 
+## Custom request reason prompts
+
+As of right now it's only possible specify a custom prompt for all or none of the Access Requests
+for a particular user. A custom request prompt can be specified by assigned the user a role with
+`spec.options.request_prompt` set to non-empty string.
+
+The ability to specify custom prompts for particular roles/requests is traced in [this
```

Comment:
We don't usually add issue links to the docs. What's the reason for doing it here?

Reply:
No particular reason. I thought it may be a good entrypoint for customers to upvote or add a comment they would like to have it. I'm happy to get rid of that.

Reply:
Would it be possible to remove this section? As Paul mentioned, we don’t typically use the documentation to share roadmap items.

The main reason is that this information will likely change once [this issue](https://github.com/gravitational/teleport/issues/55544) is resolved, and there’s a risk we might forget to update or remove this reference from the docs later on.

Reply:
Removed.